server/routes/referrals/complexityLevelPresenter.test.ts

```diff
@@ -61,15 +39,15 @@ describe('ComplexityLevelPresenter', () => {
     describe('when the referral already has a selected complexity level', () => {
       it('sets checked to true for the referral’s selected complexity level', () => {
         draftReferral.complexityLevelId = '110f2405-d944-4c15-836c-0c6684e2aa78'
-        const presenter = new ComplexityLevelPresenter(draftReferral, socialInclusionComplexityLevels)
+        const presenter = new ComplexityLevelPresenter(draftReferral, serviceCategory)
 
         expect(presenter.complexityDescriptions.map(description => description.checked)).toEqual([false, true, false])
       })
     })
 
     describe('when there is user input data', () => {
       it('sets checked to true for the complexity level that the user chose', () => {
-        const presenter = new ComplexityLevelPresenter(draftReferral, socialInclusionComplexityLevels, null, {
+        const presenter = new ComplexityLevelPresenter(draftReferral, serviceCategory, null, {
           'complexity-level': '110f2405-d944-4c15-836c-0c6684e2aa78',
```

Comment:
To decrease dependence on the exact values in the factory

```suggestion
          'complexity-level': serviceCategory.complexityLevels[1],
```